notebooks/experimental/prediction_tensorflow_model.ipynb

```diff
@@ -40,58 +48,108 @@
+      "Requirement already satisfied: jupyterlab-pygments in /home/ksenia-evin/.local/lib/python3.8/site-packages (from nbconvert->notebook>=4.4.1->widgetsnbextension~=3.0.0->ipywidgets>=7.0.0->unfolded.map-sdk==0.3.0) (0.1.2)\n",
+      "Requirement already satisfied: bleach in /home/ksenia-evin/.local/lib/python3.8/site-packages (from nbconvert->notebook>=4.4.1->widgetsnbextension~=3.0.0->ipywidgets>=7.0.0->unfolded.map-sdk==0.3.0) (4.0.0)\n",
+      "Requirement already satisfied: defusedxml in /home/ksenia-evin/.local/lib/python3.8/site-packages (from nbconvert->notebook>=4.4.1->widgetsnbextension~=3.0.0->ipywidgets>=7.0.0->unfolded.map-sdk==0.3.0) (0.7.1)\n",
+      "Requirement already satisfied: entrypoints>=0.2.2 in /usr/lib/python3/dist-packages (from nbconvert->notebook>=4.4.1->widgetsnbextension~=3.0.0->ipywidgets>=7.0.0->unfolded.map-sdk==0.3.0) (0.3)\n",
+      "Requirement already satisfied: nbclient<0.6.0,>=0.5.0 in /home/ksenia-evin/.local/lib/python3.8/site-packages (from nbconvert->notebook>=4.4.1->widgetsnbextension~=3.0.0->ipywidgets>=7.0.0->unfolded.map-sdk==0.3.0) (0.5.4)\n",
+      "Requirement already satisfied: testpath in /home/ksenia-evin/.local/lib/python3.8/site-packages (from nbconvert->notebook>=4.4.1->widgetsnbextension~=3.0.0->ipywidgets>=7.0.0->unfolded.map-sdk==0.3.0) (0.5.0)\n",
+      "Requirement already satisfied: mistune<2,>=0.8.1 in /home/ksenia-evin/.local/lib/python3.8/site-packages (from nbconvert->notebook>=4.4.1->widgetsnbextension~=3.0.0->ipywidgets>=7.0.0->unfolded.map-sdk==0.3.0) (0.8.4)\n",
+      "Requirement already satisfied: pycparser in /home/ksenia-evin/.local/lib/python3.8/site-packages (from cffi>=1.0.0->argon2-cffi->notebook>=4.4.1->widgetsnbextension~=3.0.0->ipywidgets>=7.0.0->unfolded.map-sdk==0.3.0) (2.20)\n",
+      "Requirement already satisfied: webencodings in /home/ksenia-evin/.local/lib/python3.8/site-packages (from bleach->nbconvert->notebook>=4.4.1->widgetsnbextension~=3.0.0->ipywidgets>=7.0.0->unfolded.map-sdk==0.3.0) (0.5.1)\n",
+      "Requirement already satisfied: packaging in /home/ksenia-evin/.local/lib/python3.8/site-packages (from bleach->nbconvert->notebook>=4.4.1->widgetsnbextension~=3.0.0->ipywidgets>=7.0.0->unfolded.map-sdk==0.3.0) (21.0)\n",
+      "Requirement already satisfied: nest-asyncio in /home/ksenia-evin/.local/lib/python3.8/site-packages (from nbclient<0.6.0,>=0.5.0->nbconvert->notebook>=4.4.1->widgetsnbextension~=3.0.0->ipywidgets>=7.0.0->unfolded.map-sdk==0.3.0) (1.5.1)\n"
      ]
     }
    ],
    "source": [
-    "!pip install pandas==1.3.2 numpy==1.19.5 sklearn==0.0 geopy==2.2.0 scipy==1.7.1 seaborn==0.11.2 matplotlib==3.4.3 tensorflow==2.6.0"
+    "!pip install pandas==1.3.2 numpy==1.19.5 sklearn==0.0 geopy==2.2.0 scipy==1.7.1 seaborn==0.11.2 matplotlib==3.4.3 tensorflow==2.6.0 unfolded.map-sdk==0.3.0"
```

Comment:
I tried running it in Amazon SageMaker and for some of the packages only earlier versions were available, so I had to downgrade. The notebook seemed to work fine with the downgraded versions.
```suggestion
    "!pip install pandas==1.1.5 numpy==1.19.5 sklearn==0.0 geopy==2.2.0 scipy==1.5.4 seaborn==0.11.2 matplotlib==3.3.4 tensorflow==2.6.0 unfolded.map-sdk==0.3.0"
```

Reply:
I was going to suggest taking out the pinned versions entirely. For one, if someone runs this in their own environment, it could break their environment by changing versions. And in a few months these pinned versions will be out of date.

I think better would be to remove pins but have a note "We tested this notebook with these package versions..."

Reply:
I am convinced that we need to freeze the version, e.g. for people without any environment and experience work with notebook.

Reply:
I can leave a comment above this cell, for example: "Run this cell only when you don't have an enviroment"

Reply:
The issue is that for new users, they'll likely be running Jupyter in their global environment. And if they have pandas 1.3.0 installed and we have them run `!pip install pandas==1.1.5` then it could break other environments they have by downgrading their pandas to 1.1.5.

Reply:
Instead of a strict pin we could have broader pins:
```
pandas>=1.0 numpy>=1.0 scipy>=1.0
```
etc.

These libraries haven't changed their APIs in a backward incompatible way in a very long time, so unless we use a specific API added in a later version having an open ended (or until the next major version: `pandas>=1.0,<2.0`) makes sense.